gradle.properties

```diff
@@ -0,0 +1 @@
+org.gradle.jvmargs=-Xmx1536M
```

Comment:
Needed to avoid To run dex in process, the Gradle daemon needs a larger heap. message in Travis.